configure

```diff
@@ -662,6 +662,7 @@ infodir
 docdir
 oldincludedir
 includedir
+runstatedir
```

Comment:
This is pretty in-the-weeds for such a simple change, but in the future, know that you shouldn't check in the `runstatedir` stuff that gets added. Try using e.g. `git add -p` and not staging it.

See #1769 for a discussion of the history of this.